docs/sources/guides/whats-new-in-v6-0.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "What's New in Grafana v6.0"
+description = "Feature & improvement highlights for Grafana v6.0"
+keywords = ["grafana", "new", "documentation", "6.0"]
+type = "docs"
+[menu.docs]
+name = "Version 6.0"
+identifier = "v6.0"
+parent = "whatsnew"
+weight = -11
++++
+
+# What's New in Grafana v6.0
+
+This update to Grafana introduces a new way of exploring your data, support for log data and tons of other features.
+
+The main highlights are:
+
+- The new query-focused [Explore]({{< relref "#explore" >}}) workflow for troubleshooting and/or for data exploration.
+- [Support for Grafana Loki]({{< relref "#explore-and-grafana-loki" >}}) - a new open source log aggregation system from Grafana Labs.
+- [Easily Switch Visualization with the Panel Edit UX Update]({{< relref "#easily-switch-visualization-with-panel-edit-ux-update" >}})
+- [Google Stackdriver Datasource]({{< relref "#google-stackdriver-datasource" >}}) is out of beta and is officially released.
+- The [Azure Monitor]({{< relref "#azure-monitor-datasource" >}}) plugin is ported from being an external plugin to being a core datasource
+
+### Explore
+
+Grafana's dashboard UI is all about building dashboards for visualization. **Explore** strips away all the dashboard and panel options so that you can focus on the query. Iterate until you have a working query and then think about building a dashboard.
+
+For infrastructure monitoring and incident response, you no longer need to switch to other tools to debug what went wrong. **Explore** allows you to dig deeper into your metrics and logs to find the cause. Grafana's new logging datasource, [Loki](https://github.com/grafana/loki) is tightly integrated into Explore and allows you to correlate metrics and logs by viewing them side-by-side.
+
+{{< docs-imagebox img="/img/docs/v60/explore_split.png" class="docs-image--no-shadow" caption="Screenshot of the new Explore option in the panel menu" >}}
+
+**Explore** is a new paradigm for Grafana. It creates a new interactive debugging workflow that integrates two pillars of observability - metrics and logs.
+
+#### Explore and Prometheus
```

Comment:
I suggest another section `Explore and other datasources` or similar to explain that you can use Explore with all the core datasources. Also maybe something about that plugin datasources can add support for this as well (plugin documentation lacking for this I suppose?)

Reply:
made that change now